src/model/graph.ts

```diff
@@ -101,5 +103,66 @@ export class Graph implements GraphInterface {
     }
   }
 
+  toProtobuf(): protobuf.Graph {
+    let edges: { [key: string]: protobuf.Edge } = {};
+    Object.entries(this.edges).forEach(
+      (entry) => (edges[entry[0]] = entry[1].toProtobuf())
+    );
+    let nodes: { [key: string]: protobuf.Node } = {};
+    Object.entries(this.nodes).forEach(
+      (entry) => (nodes[entry[0]] = entry[1].toProtobuf())
+    );
+    let participants: { [key: string]: protobuf.Participant } = {};
+    Object.entries(this.participants).forEach(
+      (entry) =>
```

Comment:
`(entry) => {}` can also be written as `([key, value]) =>`. In that case, you no longer need `entry[1]`, but can use the `key` and `value` variables instead. This makes the code much easier to read.